modules/snooker/src/main/ts/ephox/snooker/api/Generators.ts

```diff
@@ -166,12 +169,53 @@ const merging = function (generators: Generators) {
       // Remove any width calculations because they are no longer relevant.
       Css.remove(raw, 'width');
       Css.remove(cell, 'width');
+
+      scope.each((attribute) => Attribute.set(raw, 'scope', attribute));
+
       return raw;
     };
   };
 
+  const merge = (cells: SugarElement[]) => {
+    const getScopeProperty = () => {
+      const attributeOpts = Arr.map(cells, (cell) => {
+        const attributeOpt = Attribute.getOpt(cell, 'scope');
+
+        return attributeOpt.map((attribute) => attribute.substr(0, 3)); // Attribute can be col, colgroup, row, and rowgroup.
```

Comment:
```suggestion
        return Attribute.getOpt(cell, 'scope').map((attribute) => attribute.substr(0, 3)); // Attribute can be col, colgroup, row, and rowgroup.
```
maybe?

Reply:
I'd agree having this as a separate variable is overly verbose and this should be inlined.